tools/lambda-promtail/lambda-promtail/main.go

```diff
@@ -128,6 +129,9 @@ func checkEventType(ev map[string]interface{}) (interface{}, error) {
 }
 
 func handler(ctx context.Context, ev map[string]interface{}) error {
+
+	fmt.Println("processing new event: %s\n", ev)
```

Comment:
`fmt.Println()` doesn't do formatting, it just prints all its arguments concatinated with a space and adds a newline to the end:
```suggestion
	fmt.Println("processing new event: ", ev)
```